Packs/MandiantAdvantageAttackSurfaceManagement/Integrations/AttackSurfaceManagement/AttackSurfaceManagement.py

```diff
@@ -0,0 +1,458 @@
+
+        response = self._http_request('GET', endpoint,
+                                     headers={'PROJECT_ID': str(self.project_id)})
+
+        if not isinstance(response, dict) or not response.get('success'):
+            raise DemistoException('The ASM API was unable to return notes for'
+                                   f'{resource_type} {resource_id} in project {self.project_id}')
+
+        return response.get('result') or []
+
+
+''' HELPER FUNCTIONS '''
+
+def helper_create_incident(issue: dict, project_id: int) -> dict:
+    severity_map = {1: 4, 2: 3, 3: 2, 4: 1, 5: 0.5}
+
+    raw_json = issue
+    raw_json['mirror_direction'] = 'Both'
+    raw_json['mirror_instance'] = demisto.integrationInstance()
+
+    issue = {
+        'name': issue.get('summary', {}).get('pretty_name'),
+        'details': json.dumps(issue, indent=4),
+        'occurred': parser.parse(issue.get('first_seen')).isoformat(),
+        'severity': severity_map.get(issue.get('summary', {}).get('severity'),),
+        'rawJSON': json.dumps(raw_json),
+        'dbotMirrorId': issue.get('id', issue.get('uid')),
+        'CustomFields': {
+            'masmcollection': issue.get('collection'),
+            'masmproject': str(project_id),
+            'masmconfirmed': issue.get('summary', {}).get('confidence') == 'confirmed',
+            'masmentityname': issue.get('entity_name'),
+            'masmcategory': issue.get('summary', {}).get('category')
+        }
+    }
+
+    return issue
+
+def fetch_incident_helper(client: Client, last_run: dict) -> typing.Tuple[dict, list]:
+    last_start_time: datetime
+
+    params = demisto.params()
+
+    if not client.project_id:
+        raise DemistoException('Must configure a Project ID to fetch incidents from Mandiant ASM')
+
+    if last_run and 'start_time' in last_run and last_run['start_time']:
+        last_start_time = parser.parse(last_run.get('start_time'))
+    else:
+        parsed_time = dateparser.parse(params.get("first_fetch"), settings={'TIMEZONE': 'UTC', 'RETURN_AS_TIMEZONE_AWARE': True})
+        if not parsed_time:
+            parsed_time = datetime.now()
+        last_start_time = parsed_time
+
+    issues = client.get_issues_since_time(last_start_time, params.get('minimum_severity'))
+    most_recent_update = None
+
+    parsed_issues = []
+
+    for issue in issues:
+        incident_details = client.get_issue_details(issue["id"])
+        new_incident = helper_create_incident(incident_details, client.project_id)
+        new_incident['dbotMirrorLastSync'] = datetime.fromtimestamp(0).strftime("%Y-%m-%dT%H:%M:%SZ")
+        new_incident['lastupdatetime'] = new_incident['dbotMirrorLastSync']
+
+        parsed_issues.append(new_incident)
+
+    parsed_issues = filter_incidents_by_duplicates_and_limit(
+        incidents_res=parsed_issues, last_run=last_run, fetch_limit=client.limit, id_field='dbotMirrorId'
+    )
+
+    most_recent_update = parsed_issues[-1]['last_seen']
```

Comment:
what is there isn't any issue? wouldn't you fail on `IndexError`?

also use the `get` method of dict.